src/__tests__/conrec.test.ts

```diff
@@ -5,10 +5,11 @@ import { convert } from 'jcampconverter';
 import { Conrec } from '..';
 
 const data = fs.readFileSync(`${__dirname}/data/zhmbc_0.jdx`, 'utf8');
-const parsed = convert(data, { noContour: true }).flatten[0];
+const parsed: any = convert(data, { noContour: true }).flatten[0];
```

Comment:
Can you remove the any?

Reply:
@lpatiny Is this a bug in the type definitions of `jcampconverter`?

Reply:
jcampconverter is not in typescript and indeed this is missing. I hope this fix this issue: https://github.com/cheminfo/jcampconverter/commit/b76e834ea1234dda0e01529bc4db374266ec56e9
(version 9.1.1)

Reply:
fixed thank you